sig-and-wg/sig-core/proposals/installer-release.md

```diff
@@ -0,0 +1,31 @@
+# Kyma Installer with embedded Kyma charts and Tiller
+
+Created on 2018-09-19 by Piotr Bochyński (@pbochynski).
+
+## Status
+
+Proposed on 2018-09-19
+
+## Context
+
+Current installation process involves downloading Kyma charts from the cloud storage. Such solution allows to release Kyma installer and kyma charts independently. In the consequence, there is possibility to install different versions (or flavors) of Kyma components using a single installer binary. It seems to be a convenient feature but in reality, it introduces the following problems:
```

Comment:
`The current installation process involves downloading Kyma charts from the cloud storage. Such solution allows to release Kyma installer and Kyma charts independently. As a consequence, there is a possibility to install different versions (or flavors) of Kyma components using a single installer binary.`

Reply:
Done